pkgs/development/python-modules/opentracing/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "opentracing";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "cfd231ba5c58f90bc277787e62861eb0c6e4af76e42957bec240bbdf71fb7e0e";
+  };
+
+#  preInstall = ''
```

Comment:
Why are these comments here?

Reply:
@alexarice I was trying to enable check phase as well. But was facing issues. That was the main reason for keeping the PR in draft.

Reply:
> I will test this in my environment and update.

Done.